pkgs/applications/science/physics/xnec2c/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+, pkg-config
+, which
+, gtk3
+, openblas
```

```suggestion
, blas
, lapack
```
That version uses the wrappers and makes it easy to switch out against other implementations.

---

Thanks!